src/store/actions.ts

```diff
@@ -1 +1,2 @@
 export { authActions } from './auth';
+export { skillActions } from './skill';
```

Comment:
Let's export all actions together
```suggestion
export * from './auth';
export * from './skill';
```